docs/content/manpages/geowave-recalcstats.txt

```diff
@@ -0,0 +1,26 @@
+//:geowave-recalcstats(1)
+//:================
+//::doctype: manpage
+
+NAME
+//:----
+
+geowave remote recalcstats - Calculate the statistics of an existing GeoWave dataset
+
+SYNOPSIS
+//:--------
+
+*geowave remote recalcstats* [options] <store name> <adapterId> <statId>
```

Comment:
I think this might be a bug ... not sure, I can't recall if you can specify statId for recalcstats or not.